src/System.Management.Automation/security/SecurityManager.cs

```diff
@@ -189,6 +185,10 @@ private bool CheckPolicy(ExternalScriptInfo script, PSHost host, out Exception r
 #endif
             if (_executionPolicy == ExecutionPolicy.Unrestricted)
             {
+                // Product binaries are always trusted
+                if (SecuritySupport.IsProductBinary(path))
```

Comment:
If the execution policy is unrestricted, then I think we don't need this check anymore.

Reply:
Chatted with Travis offline. This check is needed to bypass files downloaded from internet to `$PSHome\Module`.
Need to add a comment about this.